src/hooks.ts

```diff
@@ -0,0 +1,17 @@
+/** @type {import('@sveltejs/kit').GetSession} */
+export const getSession: import("@sveltejs/kit").GetSession = async () => {
+	const events = await Promise.all(
```

Comment:
rather than put this in `getSession` I'd put it in an endpoint (https://kit.svelte.dev/docs#routing-endpoints) that can be called from the `load` method in `events/index.svelte`

Reply:
Ok, I'll try this out, thank you 👍

Reply:
f664df7